src/objects.js

```diff
@@ -1,41 +1,68 @@
 const createPerson = (name, age) => {
   // your code here
+  const person = new Object();
+  person.name = name, 
+  person.age = age;
+  
+  return person;
 };
 
 const getName = object => {
   // your code here
+  return object.name;
 };
 
 const getProperty = (property, object) => {
   // your code here
+  return object[property];
 };
 
 const hasProperty = (property, object) => {
   // your code here
+  if (object[property]) {
+    return true;
+  }
+  return false;
 };
 
 const isOver65 = person => {
   // your code here
+  if (person.age > 65) {
+    return true;
+  }
+  return false;
```

Comment:
You could simply return the result of evaluating the condition since you are only looking for a boolean value here:
```
  return person.age > 65;
```